qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,154 @@
+    Given https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And User should see only 1 vacancy "Data Engineer (SQL)"
+
+Scenario: 1.1_07 Checking the link with the selected components and technologies 
+    Given https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User Copies link with filters https://career.quantori.com/ru?competences=Databases&technologies=SQL
+    And User Opens a new page in his browser 
+    And pastes the copied link
+    Then User should be redirected to the new page https://career.quantori.com/ru?competences=Databases&technologies=SQL
+    And User should see only 1 vacancy "Data Engineer (SQL)"
+
+Scenario: 1.1_08 Checking the form "Open job positions" without filters
+    Given https://career.quantori.com/ru/positions website is opened 
+    When User pushes "Open job positions" form
+    Then User should stay on the same page
+    And User should see default page with positions 
+
+Scenario: 1.1_09 Checking the form "Open job positions" with filters
+    Given https://career.quantori.com/ru/positions website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User pushes "Open job positions" form
+    Then User should be redirected to the new page
+    And  User should see default page "Quantori open job positions" without filters
+
+Scenario: 1.1_10 Saving the cache when the page is refreshed on main Quantori page
+    Given https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User refreshes page
+    Then User should see default page with filters
+
+Scenario: 1.1_11 Saving the cache when the page is refreshed on "Quantori open job positions" page
+    Given  https://career.quantori.com website is opened 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User refreshes page
+    Then User should see https://career.quantori.com/ru/positions?competences=Databases&technologies=SQL page with filters
+
+Scenario: 1.1_12  Checking the checkboxes when selecting a competency
+    Given https://career.quantori.com website is opened 
```

Comment:
you have already written this Given in the Background

Reply:
corrected